assembl/static2/js/app/utils/hashLinkScroll.js

```diff
@@ -15,7 +16,7 @@ const getContentAreaOffset = () => {
   return offset;
 };
 
-export const scrollToPost = (postElement, smooth = true) => {
+export const scrollToPost = (postElement: any, smooth: boolean = true) => {
```

Comment:
I think postElement must be a React.Node

Reply:
@paolina92 you forgot this comment.